src/ImageGallery.jsx

```diff
@@ -582,11 +582,15 @@ export default class ImageGallery extends React.Component {
 
             onTouchStart={event => this.slideToIndex.call(this, index, event)}
             onClick={event => this.slideToIndex.call(this, index, event)}>
-
-            <img
-              src={item.thumbnail}
-              alt={item.thumbnailAlt}
-              onError={onThumbnailError.bind(this)}/>
+            <div style={{ width: '63px' }}>
+              <img
+                src={item.thumbnail}
+                alt={item.thumbnailAlt}
+                onError={onThumbnailError.bind(this)}/>
+               <span style={{ fontSize: '0.8em', color: 'black' }}>
+                 {item.label}
```

Comment:
This should be consistent with the other thumbnail properties being called `thumbnailLabel`